### PR TITLE
FIX: to neural assessment and BXT

### PR DESCRIPTION
**Pull request by stnava**
also - need to update the assessment models as they have `None` instead of `False`

```python

mdl=tf.keras.models.load_model(mfn, compile=False)
for k in range( len(mdl.layers) ):
  if mdl.layers[k].trainable is None:
    mdl.layers[k].trainable=False
tf.keras.models.save_model( mdl, mfn )

```

will send those models later

the bxt fix just thresholds the sigmoid function.

### CODE REVIEW

**Comment by ntustison** [comment]
Nice.  I assume we’ll have to make the changes in ANTsRNet as well, correct?

**Comment by stnava** [comment]
I think the code for assessment is the same or almost the same.    the sigmoid change - probably yes assuming  we like it.   the models all need to be updated if they are to be used with newer tf versions.  

this model layer issue may appear again elsewhere.

**Comment by ntustison** [comment]
Great.  Let me know if you need me to make any fixes (ANTsRNet or elsewhere).  I ran across a couple of tf issues when getting my deep_flash training scripts to work on archeozoic.

**Comment by stnava** [comment]
couple comments:

* repaired models `/raid/data_BA/fixed_models`
* greatly simplified global quality prediction - I am using augmentation ( random rotation ) to get around orientation issues --- via wrapper function

**Comment by ntustison** [comment]
@stnava --- Is the issue that the old models were incompatible with tensorflow.keras?

**Comment by stnava** [comment]
this is no longer allowed `mdl.layers[k].trainable is None `

**Comment by stnava** [comment]
am merging in order to help push some other things forward.
